src/content/app/tools/blast/components/blast-job-submit/BlastJobSubmit.tsx

```diff
@@ -15,17 +15,39 @@
  */
 
 import React from 'react';
-import noop from 'lodash/noop';
+import { useSelector } from 'react-redux';
 
 import { PrimaryButton } from 'src/shared/components/button/Button';
 
+import { getBlastFormData } from '../../state/blast-form/blastFormSelectors';
+import { BlastFormState } from '../../state/blast-form/blastFormSlice';
+
 const BlastJobSubmit = () => {
   // TODO:
   // 1) validate the blast form before enabling the button
   // 2) actually do the job submission
+  const blastFormData = useSelector(getBlastFormData);
+
+  const onBlastSubmit = () => {
+    const postData = createBlastSubmissionData(blastFormData);
+    return postData;
+  };
+
+  const createBlastSubmissionData = (blastFormData: BlastFormState) => {
+    return {
+      email: 'blast2020@ebi.ac.uk',
```

Comment:
How do we deal with this on the current Ensembl sites? Do we submit a fake email? What does web-prod say about this?

Reply:
`ensembl-webteam@ebi.ac.uk`
https://github.com/Ensembl/ebi-plugins/blob/master/common/conf/SiteDefs.pm#L121
Webprod says it is mandatory and should be valid. I am not sure where this is used.

Reply:
I guess the email address will receive the job status/results page URL.

Reply:
If this is a fake/unimportant/not user's email anyway, shouldn't it be added to the payload on the server side? Any reason to hard-code an email address on the client?

Reply:
Yes, it can be added in BE side for now.

Reply:
Yes we should move it to BE. Email is used by Jobdispatcher for LSF queue priority decision.